### DISCUSSION TITLE
How to disable auto-updating(serverless framework 4.1.12)

### DISCUSSION THREAD

**Discussion by shiki-sou** [serverless/serverless#12683] [Q&A] [answered]
From link below we found that serverless framework v4 is auto-updating every 24 hours.
https://www.serverless.com/framework/docs/getting-started#update-serverless-framework

Is there any setting that we could disable auto-updating?

**Comment by rbro112** [comment]
This is a pretty heavy-handed feature. Being unable to pin to a specific (working) version has us stuck in an infinite-breaking loop, as 4.1.13 has seemingly introduced a breaking change (https://github.com/serverless/serverless/issues/12689). Since we cannot pin to the last working version (4.1.12), we are entirely stuck being unable to get back to a working state.

**Comment by austencollins** [comment, marked as answer]
We're working on this now. Please follow along here: https://github.com/serverless/serverless/issues/12692
